script.js

```diff
@@ -0,0 +1,124 @@
+//First Prompt to Pick a Number>Character
+
+startGame();
+
+function startGame(){
+    var start_number = prompt("Good morrow, my fair midsummer adventurer. Please pick a number 1 through 4 (i.e. 1, 2, 3, or 4):");
+    var start_integer = parseInt(start_number);
+    while (start_integer == 0 || start_integer >= 5) {
+      start_number = prompt("Good morrow, my fair midsummer adventurer. Please pick a number 1 through 4 (i.e. 1, 2, 3, or 4):");
+      start_integer = parseInt(start_number);
+   }
+   playGame(start_number);
+}
+
+
+
+
+
+
+//1->Hermia
+//function startGame (start_number) {
+
+function playGame (start_number) {
+if (start_number === "1") {
```

Comment:
Remember that JavaScript is CamelCase for naming! Ruby uses snakecase for naming variables `(start_number)`. So instead of `var start_number = ` for JS it should be instead `var startNumber = `